### PR TITLE
Add requested tests for LineOfCredit.t.sol

### PR DESCRIPTION
**Pull request by cantbeevil**
* Add fuzz to requested tests
- addCredit and borrow
    - if borrow > lend → no liquidity
    - if lend ≥ borrow → success
    - if token = 0 → no price
    - if token = ETH → eth moves
    if token = token → token moves

- depositAndRepay
    - from multiple accounts
    - less than debt → liquidated after ttl
    - more than debt → REPAID

### CODE REVIEW

**Comment by kibagateaux** [comment]
tests look good. Did you try any of them with the `prove_` prefix instead of `test_`? I tried using prove in early days pre-forge but they would hang or take too long.

**Comment by cantbeevil** [comment]
> tests look good. Did you try any of them with the `prove_` prefix instead of `test_`? I tried using prove in early days pre-forge but they would hang or take too long.

not sure what that is. i looked it up but didn't find anything

File: contracts/tests/LineOfCredit.t.sol [hunk 1]
```diff
@@ -309,46 +313,50 @@ contract LineTest is Test{
         assertEq(lender.balance, mintAmount - 1 ether, "Lender should have initial mint balance minus 1e18");
     }
 
-    function test_can_borrow() public {
+    function test_can_borrow(uint amount) public {
```
[the remark this answers is not in the document]
Comment (cantbeevil) [reply contracts/tests/LineOfCredit.t.sol]: done

File: contracts/tests/LineOfCredit.t.sol [hunk 2]
```diff
@@ -705,4 +713,101 @@ contract LineTest is Test{
...
+        supportedToken1.approve(address(line), repayAmount);
+        line.depositAndRepay(repayAmount);
+        vm.stopPrank();
+        
+        
+        (uint256 p, uint256 i) = line.updateOutstandingDebt();
+        assertEq(p + i, 0, "Line outstanding credit should be 0");
+    }
+
+    function test_deposit_and_repay_less_than_debt_after_ttl(uint credit) public {
```
[the remark this answers is not in the document]
Comment (cantbeevil) [reply contracts/tests/LineOfCredit.t.sol]: done

**Comment by kibagateaux** [comment]
hmmm maybe its only a dapptools thing. Its for symbolic execution testing instead of fuzzing

**Pull request merged**